COMPATIBILITY.md

```diff
@@ -79,6 +79,8 @@ Newly filed issues should bear the label `pipeline` for ease of tracking.
 - [ ] `HockeyappRecorder` (`hockeyapp`): [JENKINS-33310](https://issues.jenkins-ci.org/browse/JENKINS-33310)
 - [X] `WsCleanup` (`ws-cleanup`): supported as of 0.30
 - [ ] `XCodeBuilder` (`xcode-plugin`): [JENKINS-33355](https://issues.jenkins-ci.org/browse/JENKINS-33355)
+- [ ] `Release Plugin` (`release`): [JENKINS-40765](https://issues.jenkins-ci.org/browse/JENKINS-40765)
+- [ ] `M2 Release Plugin` (`m2release`): [JENKINS-40766](https://issues.jenkins-ci.org/browse/JENKINS-40766)
```

Comment:
thanks for the feedback!
i moved both plugins to the custom step section, according to the proposals:
- https://github.com/jenkinsci/release-plugin/pull/22
- https://github.com/jenkinsci/m2release-plugin/pull/33